styles.css

```diff
@@ -0,0 +1,150 @@
+*{
+	box-sizing: border-box;
+}
+
+html{
+	font-size: 20px;
+}
+
+/* centers the webpage */
+body{
+	display: flex;
+	justify-content: center;
+}
+
+body > div{
+	width: 1500px;
+}
+
+
+/* navbar section*/
+nav{
+	display: flex;
+	background: #E5E2DE;
+	height: 50px;
+	margin: 10px 0;
+	justify-content: space-between;
+	align-items: center;
```

Comment:
this doesn't change anything on the inspector when un-ticked

Reply:
I think it does do something but it wasn't obvious before because the height was so small.  I made the height larger so it should be clear now.